Makefile-test.am

```diff
@@ -534,6 +535,20 @@ test_unit_tcti_spi_helper_LDADD   = $(CMOCKA_LIBS) $(libtss2_tcti_spi_helper)
 test_unit_tcti_spi_helper_SOURCES = test/unit/tcti-spi-helper.c
 endif
 
+test_unit_tcti_spi_lt2go_CFLAGS  = $(CMOCKA_CFLAGS) $(TESTS_CFLAGS)
```

Comment:
This block needs to be in the AM conditional block as well:
```
if ENABLE_TCTI_SPI_LT2GO
...
endif
```

Reply:
fixed